src/components/scene/stats.js

```diff
@@ -11,6 +11,11 @@ var AFrameStats = window.aframeStats;
  * Stats appended to document.body by RStats.
  */
 module.exports.Component = registerComponent('stats', {
+  schema: {
+    type: 'boolean',
```

Comment:
I believe there's a bug associated to this. Can you try the following with your patch:
```<a-scene stats=""></a-scene>``` Do the stats show up?